### DISCUSSION TITLE
file downloading error at installing phase because of an additional :

### DISCUSSION THREAD

**Discussion by Eren-0X** [projectdiscovery/dnsx#556] [Q&A]
**#bug**
hello
i got error in installing dnsx at ubuntu
it downloads github.com/pierrec/lz4 v2.6.1+incompatible
and when it reads it from https://proxy.golang.org/github.com/pierrec/lz4/@v/v2.6.1+incompatible.zip: it results an error because it has an additional : at the end of link (when i deleted it in browser, it get downloaded)

![bug](https://github.com/projectdiscovery/dnsx/assets/149108455/2630dced-ac71-4afb-ae01-9b23b8b09a50)

**Comment by dogancanbakir** [comment]
Thanks for opening the issue. Could you please let me know which version of Golang is installed? Make sure you've go version 1.21

**Reply by Eren-0X** [reply]
as told in installation section, i installed it with go1.19, not with 1.21
if it has issue with go1.19, please update the github page.

thanks a lot !!!
